indico/modules/rb/client/js/common/rooms/edit/RoomEditModal.jsx

```diff
@@ -102,6 +104,17 @@ function RoomEditModal({roomId, locationId, onClose, afterCreation}) {
     );
   }, [roomId]);
 
+  useEffect(() => {
+    (async () => {
+      if (!roomNameFormat) {
```

Comment:
Is this check really correct? If the locationId changed for some reason (probably never happens) we *would* need to fetch the format again